supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/api/GcpBatchRequestFactoryImpl.scala

```diff
@@ -119,7 +119,7 @@ class GcpBatchRequestFactoryImpl()(implicit gcsTransferConfiguration: GcsTransfe
     } else {
       Disk.newBuilder
     }
-    disk.setImage("batch-debian")
```

Comment:
This was done to support #8 (see PR description). Need to be careful with unintended side effects of reverting that change.

Looking at the code above, you can get a similar effect by explicitly specifying `batch-cos` as the boot disk in the task's runtime attributes.